src/components/responsive/ResponsiveContainer.js

```diff
@@ -13,27 +13,33 @@ class ResponsiveContainer extends Component {
         return isSSR ? Responsive.onlyTablet.minWidth : window.innerWidth;
     }
 
-    userInSignupFlow = () => {
+    showNavLinks = () => {
+        const { account, location } = this.props;
+        const accounts = account.accounts;
         const signUpRoutes = ['create', 'set-recovery', 'setup-seed-phrase'];
-        const currentBaseRoute = this.props.location.pathname.replace(/^\/([^\/]*).*$/, '$1');
-        return signUpRoutes.includes(currentBaseRoute);
+        const currentBaseRoute = location.pathname.replace(/^\/([^\/]*).*$/, '$1');
+
+        if (signUpRoutes.includes(currentBaseRoute))
```

Comment:
nit: let's always use `{}` brackets for statements if it's not a one-liner (e.g. `if (someThing) return anotherThing;`). Otherwise it can be confusing at times.